client/my-sites/customer-home/cards/features/launch-site/style.scss

```
@@ -0,0 +1,13 @@
+.launch-site {
+	&__card-subheader {
```

Can we use explicit class names here? I know this is how we had it before, but this makes hard to search fo a specific class.
```suggestion
	.launch-site__card-subheader {
```

---

2364db9ac49b327383fc61544dceaa0bfb895e2b